commcare-support-library/src/main/java/org/commcare/commcaresupportlibrary/BiometricIdentifier.java

```diff
@@ -0,0 +1,7 @@
+package org.commcare.commcaresupportlibrary;
+
+public enum BiometricIdentifier {
```

Comment:
I don't think we wanna allow ` UNKNOWN` or  `INVALID` here.

Reply:
73d6a98c861ffa373fb7195728375b0ab06bd018